packages/@aws-cdk/aws-redshift/lib/private/database-query-provider/index.ts

```diff
@@ -18,3 +18,5 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
   }
   return subHandler(event.ResourceProperties, event);
 }
+
+export { ColumnEncoding } from './types';
```

Comment:
Can you delete these two lines? `index.ts` imports stuff from `table.ts`, and `table.ts` imports stuff from `types.ts`. My suspicion is that this is some weird dependency issue.

Reply:
I could, but I'll need to duplicate the code for `ColumnEncoding`, or not implement it in the private folder

Reply:
ah, you have this in the readme:

```

Table columns can be configured to use a specific compression encoding:

```ts fixture=cluster
import { ColumnEncoding } from '@aws-cdk/aws-redshift';
```

this means it's supposed to be public-facing. If users should be able to call it, it should not live in the private folder.

Reply:
Makes sense, I assumed I could export it out from the private folder. I'll work on changing this

Reply:
This has been removed from the `private` folder. It can still be accessed from the public, which is what is intended.